### ISSUE TITLE
CoordinateFromPixelY, CoordinateFromPixel provides incorrect data

### ISSUE THREAD

**Issue by cinis77** [ScottPlot/ScottPlot#833]
Hi,

I have noticed that CoordinateFromPixelY provides incorrect data, this bug can be seen in demo as well with mousemove event.

I am plotting my own data with plotsignal event, tried with scatter and got the same results. The data is plotted correctly as i have already tested it out.

My data
![image](https://user-images.githubusercontent.com/19565936/109126420-e2725000-7755-11eb-990b-16507ddbb461.png)

Demo
![image](https://user-images.githubusercontent.com/19565936/109126782-51e83f80-7756-11eb-930a-1685d947d74c.png)

```cs
 private void Ploter_MouseMove(object sender, MouseEventArgs e)
        {
            double mouseCoordY = e.MouseDevice.GetPosition(Ploter).Y;
            YCoordinateLabel.Text = $"{Ploter.plt.CoordinateFromPixelY(mouseCoordY):0.00000000}";
            Ploter.Render();
        }
```

I tried other demos, but demo code is done with different lib than it is in nuget from Scott WPF.

**Comment by StendProg** [comment]
Hi @cinis77,
Do you use DPI scaling on your system?
What version of ScottPlot do you use?

**Comment by cinis77** [comment]
> Hi @cinis77,
> Do you use DPI scaling on your system?
> What version of ScottPlot do you use?

No i am using autoscaling and scott 4.0 stable version

**Comment by StendProg** [comment]
> No i am using autoscaling and scott 4.0 stable version

I tried 4.0.48 and the Demo works as it should. Similar problems have already been encountered and are usually associated with enabled scaling at the operating system level. Please double check  that you have 100% scaling in Windows. 
If it does not help, I can only advise you to try 4.1.9-beta.

**Comment by swharden** [comment]
Hi @cinis77, thanks for raising this issue!

I am surprised by the appearance of the cursor in your screenshot. First, it's in the screenshot: are you using a virtual machine or something? Second, it's very large: are you sure you are not using display scaling? Both of these issues could be causes for why mouse tracking is not working as expected.

I also agree with @StendProg the best next step would be to try the latest version of ScottPlot 4.1 on NuGet (it's marked pre-release). It has improved mouse tracking, and it may work as soon as you start using it. If not, it will give even more insight into what might be causing your issue.

Looking forward to learning more!
Scott

**Comment by cinis77** [comment]
Hi,

I have downloaded scottplot 4.1.9-beta and it works perfectly without this issue. My current windows scaling is 175%. 

Thank you.

**Comment by swharden** [comment]
> I have downloaded scottplot 4.1.9-beta and it works perfectly without this issue.

That's great to hear! I probably won't work backwards trying to fix this in ScottPlot 4.0, and since 4.1 is working well I'll go ahead and close this issue. Thanks for the follow-up @cinis77!
